usbexec.py

```diff
@@ -112,7 +112,7 @@ def command(self, request_data, response_length):
 
   def execute(self, response_length, *args):
     cmd = str()
-    for i in range(len(args)):
+    for i in enumerate(args):
```

Comment:
Did you test this?  enumerate() returns à tulle.

Reply:
You're right, this PR makes no sense.

Reply:
I believe this change is a nice optimization but it’s implementation is incomplete...
```python
for i, arg in enumerate(args):
    # and then change all instances of __args[i]__ with __arg__.